enhancements/ingress/set-delete-http-headers.md

```diff
@@ -735,26 +735,12 @@ another [option](https://docs.openshift.com/container-platform/4.12/networking/i
 but have actual use-cases (which aren't covered by the existing option). They will be allowed, with notes in the API and product documentation,
 and possibly a warning on admission.
 
-For example: If someone creates an Ingress Controller with the following:
-```sh
-apiVersion: operator.openshift.io/v1
-kind: IngressController
-metadata:
-  name: default
-  namespace: openshift-ingress-operator
-spec:
-  httpHeaders:
-    forwardedHeaderPolicy: Replace
-```
-Or if someone add a route annotation `haproxy.router.openshift.io/set-forwarded-headers`,
-then the `set-header` stanzas will be added in `haproxy.config` using the [haproxy-config.template](https://github.com/openshift/router/blob/0cd9a58dff77127562f7851ccfa5cda79b8cef48/images/router/haproxy/conf/haproxy-config.template#L565-L580)
+In contrast to request headers, in which the IngressController values override the Route spec.httpHeaders.actions values, 
+for response headers the Route values override the IngressController values.
 
-However, if someone wants to set custom values for these then they need to use the API of this EP.
-
-The custom values set using `spec.httpHeaders.actions` will take over the value defined in annotation `haproxy.router.openshift.io/set-forwarded-headers` or `spec.httpHeaders.forwardedHeaderPolicy`.  
-
-Please note that the setting of forwardedHeaderPolicy happens at the plain backend section of haproxy.config i.e. per route level and not at the front-end section
-of the haproxy.config level as per the above-mentioned link.
+The custom HTTP response value of `x-forwarded` header set using `spec.httpHeaders.actions` of the IngressController will take over the value defined in annotation `haproxy.router.openshift.io/set-forwarded-headers` of Route, `spec.httpHeaders.forwardedHeaderPolicy` of Ingress Controller
+or `spec.httpHeaders.actions` of Route as in case of HTTP response headers, HAProxy overrides the value of the backend of same header name which
+was present in the frontend section.
```

Comment:
Without this enhancement, does HAProxy ever set `x-forwarded-*` in response headers?  The `x-forwarded-*` headers are request headers; given that, none of what you wrote makes any sense.

Reply:
ok. i tried to update this.